.github/workflows/test.yml

```diff
@@ -6,8 +6,7 @@ on:
       - main
 
 jobs:
-  test_and_lint:
-    name: test&lint
+  lint:
```

Comment:
There's no need to run lint and pack on Windows as well. Pack actually seems to be really slow on Windows.